lib/cloud_controller/paging/sequel_paginator.rb

```diff
@@ -2,6 +2,7 @@
 
 module VCAP::CloudController
   class SequelPaginator
+    EXCLUDE_FROM_PAGINATION_WITH_WINDOW_FUNCTION = [:events].freeze
```

Comment:
It might be good to add a comment here; something like: "Don't use window function for the 'events' table as this table might contain millions of rows which can lead to a performance degradation."